src/mesh/simplemesh.jl

```diff
@@ -25,6 +25,40 @@ julia> mesh   = SimpleMesh(points, connec)
 See also [`Topology`](@ref), [`GridTopology`](@ref),
 [`HalfEdgeTopology`](@ref), [`SimpleTopology`](@ref).
 
+
+We can construct standard parametric surfaces from simpler meshes
+
+
+```julia
+function ϕ(u,v)
+
+  x = 5/4*(1 - v/2π )*cos(2v)*(1 + cos(u)) + cos(2*v)
+  y = 5/4*(1 - v/2π )*sin(2v)*(1 + cos(u)) + sin(2*v)
+  z = 10*v/2π +5/4*(1 - v/2π )*sin(u) + 15
+  return x,y,z
+
+end
+
+# Parametric Space
+us = LinRange(0,2π,10)[1:end-1] #this component is periodic
+vs = LinRange(0,π,40)
+
+periodic = (false, true)
+vertex = [(Point∘ϕ)(u,v)...) for u in us for v in vs]
+topo = GridTopology( (length(vs)-1 , length(us)-1) .+ periodic, periodic)
+mesh = SimpleMesh(vertex, topo)
+viz(mesh, showfacets = true)
+```
+
+We can also use it to construct meshes from more complicated meshes
+
+```julia
+f(p) = p + rand(3) * 0.25
+mesh_noisy = SimpleMesh([(Point∘f)(p) for p in coordinates.(vertices(mesh))], topology(mesh) )
+viz(mesh_noisy)
```

Comment:
please follow the code style of the other examples nearby. We do not use underscores, the name vertex is singular but the variable stores a list of vertices, ...